pkg/controller/controller.go

```diff
@@ -62,6 +63,9 @@ func (c *Controller) renderGitHubLabels() (github.ResultLabels, error) { //nolin
 	if labels.NoChangesLabelColor == "" {
 		labels.NoChangesLabelColor = "0e8a16" // green
 	}
+	if labels.NoChangesInInfraLabelColor == "" {
+		labels.NoChangesInInfraLabelColor = "1d76db" // blue (same as add-or-update for backward compatibility)
```

Comment:
the default value is set to the same one as `add-or-update`

Reply:
As I mentioned, I think the value should be same with `no-changes`.

- https://github.com/suzuki-shunsuke/tfcmt/pull/1871#issuecomment-3172925609

I'm aware that the compatibility is important, but at the same time the default value should be smart.
tfcmt works without a configuration file with smart default.
I think `no-changes` is desirable for many users.
I think tools depending on this compatibility aren't so many.
Main purpose of these labels are for human rather than automation, meaning people can understand changes (no change, add-or-update, or destroy) at glance.

Reply:
Oh I'm sorry for not reading your comment carefully. I'll fix it.